src/components/ForecastSummary.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "../styles/App.css";
+import "../styles/ForecastSummary.css";
+import iconData from "../data/iconData.json";
+
+function ForecastSummary(props) {
+  const { date, description, icon, temperature, onSelect } = props;
+  const formattedDate = new Date(date).toDateString();
+  const weatherCode = icon;
+  // `${icon.slice(0, 1)}00`;
+  return (
+    <div className="forecast-summary" data-testid="forecast-summary">
+      <div className="forecast-summary__date">{date}</div>
```

Comment:
You are rendering the unformatted date here and the formatted date one line below.
It should be:
````
<div className="forecast-summary__date">{formattedDate}</div>
`````
And line 14 should be deleted